tests/ha/filesystem.pm

```diff
@@ -61,11 +62,18 @@ sub run {
     # DLM process needs to be started
     ensure_process_running 'dlm_controld';
 
+    # ocfs2 package should be installed by default
     if ($fs_type eq 'ocfs2') {
         # Test if OCFS2 kernel module package is installed
-        die 'ocfs2 kernel package is not installed' unless is_package_installed 'ocfs2-kmp-default';
+        if (!is_package_installed 'ocfs2-kmp-default') {
+            record_info 'ocfs2 kernel package is not installed';
+            zypper_call 'in ocfs2-kmp-default';
```

Comment:
If you don't make a difference in the test result if the package is there or not I recommend then why not just call `zypper_call` regardless

Reply:
'ocfs2-kmp' package should be installed by default with the HA pattern, but in case it is not (a bug, already happen one time) I would like to have all the tests run for BV, and the information that there is an issue with the package through the `record_info`. I could use a `record_soft_failure` bug I need a bug number for that.

Reply:
ok, so what is the problem then to just create a bug and reference it? I myself wouldn't hesitate to create a bug and use that because I see our bugzilla instance as a generic reporting platform and not a work assignment platform. You can even assign the bug to yourself if you don't want to piss off anyone ;)

Reply:
I searched in my already opened bugs, because I already had the issue, and I've changed the code to record a failure with the appropriate bug number instead of the `record_info`.
Should be better now :)